cmd/internal/cmderr/checks.go

```diff
@@ -17,10 +17,10 @@ func RequirePostmanAPICredentials(explanation string) (string, error) {
 		if env.InDocker() {
 			printer.Infof("Please set the POSTMAN_API_KEY environment variables on the Docker command line.\n")
 		} else {
-			printer.Infof("Use the POSTMAN_API_KEY environment variables, or run 'postman-insights-agent login'.\n")
+			printer.Infof("Please set the POSTMAN_API_KEY environment variables, either in your shell session or prepend it to postman-insights-agent command.\n")
```

Comment:
```suggestion
			printer.Infof("Please set the POSTMAN_API_KEY environment variable, either in your shell session or prepend it to postman-insights-agent command.\n")
```

I am unsure whether the part about prepending it is clear (though I know what you mean?)  Maybe we should assume people know how to set environment variables?